<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1160</width>
    <height>783</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1160</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="MplWidget" name="MplWidget" native="true"/>
    </item>
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>7</number>
       </property>
       <property name="bottomMargin">
        <number>260</number>
       </property>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>65</height>
          </size>
         </property>
         <property name="text">
          <string>Прогноз
для встроенных рядов</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>65</height>
          </size>
         </property>
         <property name="text">
          <string>Переключиться на график
загруженного ряда</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_openDialog">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>65</height>
          </size>
         </property>
         <property name="text">
          <string>Загрузить ряд (.csv)</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>1200000</width>
        <height>235</height>
       </size>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="title">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
       </property>
       <property name="formAlignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
       <property name="horizontalSpacing">
        <number>7</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="labelTable1">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>560</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="labelTable2">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>560</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QTableWidget" name="tableWidget">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>565</width>
           <height>175</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>900</width>
           <height>180</height>
          </size>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="gridStyle">
          <enum>Qt::SolidLine</enum>
         </property>
         <property name="sortingEnabled">
          <bool>false</bool>
         </property>
         <property name="cornerButtonEnabled">
          <bool>true</bool>
         </property>
         <attribute name="horizontalHeaderCascadingSectionResizes">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderMinimumSectionSize">
          <number>49</number>
         </attribute>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>80</number>
         </attribute>
         <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
          <bool>false</bool>
         </attribute>
         <row>
          <property name="text">
           <string>Худшее
значение</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>Результат</string>
          </property>
          <property name="font">
           <font>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
         </row>
         <row>
          <property name="text">
           <string>Лучшее
значение</string>
          </property>
         </row>
         <column>
          <property name="text">
           <string>POCID</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Tendency</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>SLG</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>AFF</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>PDA</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>ARV</string>
          </property>
         </column>
         <item row="0" column="0">
          <property name="text">
           <string>0</string>
          </property>
         </item>
         <item row="0" column="1">
          <property name="text">
           <string>1</string>
          </property>
         </item>
         <item row="0" column="2">
          <property name="text">
           <string>-∞</string>
          </property>
         </item>
         <item row="0" column="3">
          <property name="text">
           <string>100</string>
          </property>
         </item>
         <item row="0" column="4">
          <property name="text">
           <string>-1</string>
          </property>
         </item>
         <item row="0" column="5">
          <property name="text">
           <string>1</string>
          </property>
         </item>
         <item row="1" column="0">
          <property name="text">
           <string/>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="background">
           <brush brushstyle="NoBrush">
            <color alpha="255">
             <red>229</red>
             <green>3</green>
             <blue>74</blue>
            </color>
           </brush>
          </property>
         </item>
         <item row="1" column="1">
          <property name="text">
           <string/>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
         </item>
         <item row="1" column="2">
          <property name="text">
           <string/>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
         </item>
         <item row="1" column="3">
          <property name="text">
           <string/>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
         </item>
         <item row="1" column="4">
          <property name="text">
           <string/>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
         </item>
         <item row="1" column="5">
          <property name="text">
           <string/>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
         </item>
         <item row="2" column="0">
          <property name="text">
           <string>100</string>
          </property>
         </item>
         <item row="2" column="1">
          <property name="text">
           <string>0</string>
          </property>
         </item>
         <item row="2" column="2">
          <property name="text">
           <string>+∞</string>
          </property>
         </item>
         <item row="2" column="3">
          <property name="text">
           <string>0</string>
          </property>
         </item>
         <item row="2" column="4">
          <property name="text">
           <string>1</string>
          </property>
         </item>
         <item row="2" column="5">
          <property name="text">
           <string>0</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QTableWidget" name="tableWidget_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>10</width>
           <height>175</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>565</width>
           <height>180</height>
          </size>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>80</number>
         </attribute>
         <row>
          <property name="text">
           <string>Худшее
значение</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>Результат</string>
          </property>
          <property name="font">
           <font>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
         </row>
         <row>
          <property name="text">
           <string>Лучшее
значение</string>
          </property>
         </row>
         <column>
          <property name="text">
           <string>POCID</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Tendency</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>SLG</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>AFF</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>PDA</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>ARV</string>
          </property>
         </column>
         <item row="0" column="0">
          <property name="text">
           <string>0</string>
          </property>
          <property name="font">
           <font>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
         </item>
         <item row="0" column="1">
          <property name="text">
           <string>1</string>
          </property>
         </item>
         <item row="0" column="2">
          <property name="text">
           <string>-∞</string>
          </property>
         </item>
         <item row="0" column="3">
          <property name="text">
           <string>100</string>
          </property>
         </item>
         <item row="0" column="4">
          <property name="text">
           <string>-1</string>
          </property>
         </item>
         <item row="0" column="5">
          <property name="text">
           <string>1</string>
          </property>
         </item>
         <item row="1" column="0">
          <property name="text">
           <string/>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
         </item>
         <item row="1" column="1">
          <property name="text">
           <string/>
          </property>
         </item>
         <item row="1" column="2">
          <property name="text">
           <string/>
          </property>
         </item>
         <item row="1" column="3">
          <property name="text">
           <string/>
          </property>
         </item>
         <item row="1" column="4">
          <property name="text">
           <string/>
          </property>
         </item>
         <item row="1" column="5">
          <property name="text">
           <string/>
          </property>
         </item>
         <item row="2" column="0">
          <property name="text">
           <string>100</string>
          </property>
         </item>
         <item row="2" column="1">
          <property name="text">
           <string>0</string>
          </property>
         </item>
         <item row="2" column="2">
          <property name="text">
           <string>+∞</string>
          </property>
         </item>
         <item row="2" column="3">
          <property name="text">
           <string>0</string>
          </property>
         </item>
         <item row="2" column="4">
          <property name="text">
           <string>1</string>
          </property>
         </item>
         <item row="2" column="5">
          <property name="text">
           <string>0</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1160</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MplWidget</class>
   <extends>QWidget</extends>
   <header>mplwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
